packages-kintone-customize/app-229-contracts/src/components/renderProjType.tsx

```diff
@@ -0,0 +1,31 @@
+
+import { createRoot } from 'react-dom/client';
+import { AutoLookup } from './AutoComplete';
+import { getProjTypes } from 'api-kintone';
+
+export default async function renderProjType(
+  initialValue:
+  { label: string, id: string }) {
```

Comment:
```suggestion
export default async function renderProjType(
  initialValue:
  { label: string, id: string },
) {
```
renderCocosumoKouji()に同じです。

Reply:
https://github.com/Lorenzras/yumecoco-monorepo/pull/186#discussion_r1164812449